### DISCUSSION TITLE
Welcome to Rsdoctor Discussions!

### DISCUSSION THREAD

**Discussion by chenjiahan** [web-infra-dev/rsdoctor#399] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AtomicScience** [comment]
Hi, is there a way to disable the opening of the Rsdoctor's page in a browser? I'd rather fix a port and just reload an already opened tab

**Reply by yifancong** [reply]
you can use  `disableclientserver` option : https://rsdoctor.dev/config/options/options#disableclientserver

**Reply by AtomicScience** [reply]
Yeah, I saw that one, doesn't it disable web-interface altogether?
